app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics.controller");
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/topics', getTopics)
+
+//err handling
+app.use((req, res, next) => {
+  if(!req.status){
+    res.status(404).send({msg: 'Route Not Found'})
+  }
```

Comment:
I dont think this actually does anything - or if it does, might find issue with it later on

i'd look at app.all() - to handle this.